### ISSUE TITLE
Question: train_t5xxl=true, generates error, finetuning

### ISSUE THREAD

**Issue by brundlesprout** [kohya-ss/sd-scripts#2197]
Using SD3 branch, experimenting with finetuning, setting the train_t5xxl issues an error. using flux_train.py module

**running the accelerate launch command bellow will initiate this error
flux_train.py: error: unrecognized arguments: --train_t5xxl**
-----------------------------------------
accelerate launch --mixed_precision bf16 --num_cpu_threads_per_process 1 flux_train.py --pretrained_model_name_or_path "./models/unet/flux1-dev.safetensors"
--clip_l "./clip/clip_l.safetensors" --t5xxl "./clip/t5xxl_fp16.safetensors"
--ae "./models/vae/ae.safetensors" --save_model_as safetensors
--sdpa --persistent_data_loader_workers --max_data_loader_n_workers 2 --seed 42
--gradient_checkpointing --mixed_precision bf16
--save_precision bf16 --config_file "./Kohya1.toml"
--learning_rate 5e-5 --max_train_epochs 60 
--save_every_n_epochs 10 --sdpa --highvram
--cache_text_encoder_outputs_to_disk --cache_latents_to_disk
--save_every_n_epochs 1 --optimizer_type adafactor
--optimizer_args "" --lr_scheduler constant_with_warmup
--max_grad_norm 0.0 --timestep_sampling shift
--discrete_flow_shift 3.1582 --model_prediction_type raw
--guidance_scale 1.0 --fused_backward_pass
--blocks_to_swap 10 --full_bf16
**--train_t5xxl`**

Thank you.

**Comment by kohya-ss** [comment]
Fine tuning is only available for FLUX.1 models. Text Encoder cannot be trained at the same time.
If you want to train Text Encoder, please do so separately using the transformers library, etc.
